src/SDKs/Search/DataPlane/Search.Tests/Tests/ExtensibleEnumTests.cs

```diff
@@ -85,7 +85,7 @@ public void CannotCreateObsoleteTypeFromValue()
             // this should result in a new TestEnumWithObsolete instance being created
             TestEnumWithObsolete obsolete = TestEnumWithObsolete.Create("Obsolete");
```

Comment:
This test no longer makes sense... Why would it ever be `DeprecatedPropertyValue` when they have different values...? At least before, the value "Obsolete" matched the name `Obsolete`... Now I'm confused as to what's being tested here.